### ISSUE TITLE
Excess of z=1 sources on tile 3526, exp 270983, 2024-12-30

### ISSUE THREAD

**Issue by sybenzvi** [desihub/desisurveyops#265]
Similar to #264, I noticed that exp 270983 on tile 3526 has an excess of redshifts at z=1 in p7. Unlike the issue raised a few days ago, I don't see a corresponding excess at z=1.6. The sky fibers from 2024-12-30 also don't look quite as odd as 2024-12-27.

I did not observe redshift pileups in other DARK tiles. We could pass the tile, but I'm marking it as unsure until we discuss both issues.

**Comment by sybenzvi** [comment]
Apologies for failing to include a screenshot:
![Screenshot 2025-01-13 at 5 43 33 PM](https://github.com/user-attachments/assets/4ea14304-14fc-48df-84d9-79a1e86ba3f0)

**Comment by geordie666** [comment]
@sybenzvi: Given that we've now discussed these issues, we could probably pass this `unsure` tile as good? I don't think this is a particularly bad excess of redshifts. 

Or do you think you'd rather request this tile to be processed without Petal 7? It's a marginal case, for me.

**Comment by schlafly** [comment]
IMO as Julien diagnosed the P7 issue as a long term feature of the z7 PSF
wings, we shouldn't mark individual tiles as bad because of it.  I'm
presuming that this is in fact due to that effect, which seems likely to
me, but I haven't demonstrated that.

**Comment by geordie666** [comment]
I agree that the z7 PSF wings are the most likely explanation, here. I'm going to reset the QA for tile 3526 and close this issue.

**Comment by geordie666** [comment]
Anand marked this tile as good, so I'm closing this issue.
